lib/hw_unique_key/hw_unique_key_acl.c

```diff
@@ -34,13 +34,15 @@ void hw_unique_key_write(enum hw_unique_key_slot kmu_slot, const uint8_t *key)
 	nrfx_nvmc_words_write(huk_addr + sizeof(huk_magic), key, HUK_SIZE_WORDS);
 	while (!nrfx_nvmc_write_done_check())
 		;
+
+	return HW_UNIQUE_KEY_SUCCESS;
 }
 
 bool hw_unique_key_is_written(enum hw_unique_key_slot kmu_slot)
 {
 	if (kmu_slot != HUK_KEYSLOT_KDR) {
 		printk("KMU slot must be HUK_KEYSLOT_KDR on nRF52840\r\n");
-		k_panic();
+		return false;
```

Comment:
What is the consequence on the logic of the users of this function for this change?

Will the write function return an error code so that it remains the same?

Reply:
The write function will indeed fail if the kmu_slot is not KDR. The write function does this check as well.
https://github.com/nrfconnect/sdk-nrf/blob/dd3b5e2c1d5144696bbd6eec55905e553632f52d/lib/hw_unique_key/hw_unique_key_acl.c#L26

 So I don't expect any behavior change in that sense.  But since now you mentioned this I realized that I the kmu_slot variable name is unfortunate I think. I will rename the variable since I do refactoring here.